src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al

```diff
@@ -186,6 +197,34 @@ table 149030 "AIT Test Suite"
             FieldClass = FlowField;
             CalcFormula = count("AIT Column Mapping" where("Test Suite Code" = field("Code")));
         }
+        field(40; "Language ID"; Integer)
+        {
+            Caption = 'Language ID';
+            TableRelation = "AIT Test Suite Language"."Language ID";
+            ValidateTableRelation = true;
+            ToolTip = 'Specifies the Windows Language ID for this test suite language.';
```

Comment:
I would not use Windows language ID, simply language that the test suite runs in/